src/runway/__init__.py

```diff
@@ -1,2 +1,2 @@
 """Set package version."""
-__version__ = '0.47.1'
+__version__ = '0.48.0'
```

Comment:
Let's drop the version change (and associated new version section in the CHANGELOG) in favor of leaving this unreleased. I think we'll make it part of 1.0

Reply:
reverted and moved changed under unreleased in changelog